db/collection_test.go

```diff
@@ -146,36 +418,36 @@ func TestCreateInstance(t *testing.T) {
 		t.Parallel()
 		db, clean := createTestDB(t)
 		defer clean()
-		collection, err := db.NewCollection(CollectionConfig{
+		c, err := db.NewCollection(CollectionConfig{
 			Name:   "Person",
 			Schema: util.SchemaFromInstance(&Person{}, false),
 		})
 		checkErr(t, err)
 
 		t.Run("WithImplicitTx", func(t *testing.T) {
 			newPerson := util.JSONFromInstance(Person{Name: "Foo", Age: 42})
-			res, err := collection.Create(newPerson)
+			res, err := c.Create(newPerson)
 			checkErr(t, err)
 			newPerson = util.SetJSONID(res, newPerson)
-			assertPersonInCollection(t, collection, newPerson)
+			assertPersonInCollection(t, c, newPerson)
 		})
 		t.Run("WithTx", func(t *testing.T) {
 			newPerson := util.JSONFromInstance(Person{Name: "Foo", Age: 42})
 			var res []core.InstanceID
-			err = collection.WriteTxn(func(txn *Txn) (err error) {
+			err = c.WriteTxn(func(txn *Txn) (err error) {
 				res, err = txn.Create(newPerson)
 				return
 			})
 			checkErr(t, err)
 			newPerson = util.SetJSONID(res[0], newPerson)
-			assertPersonInCollection(t, collection, newPerson)
+			assertPersonInCollection(t, c, newPerson)
 		})
 	})
 	t.Run("Multiple", func(t *testing.T) {
 		t.Parallel()
 		db, clean := createTestDB(t)
 		defer clean()
-		collection, err := db.NewCollection(CollectionConfig{
+		c, err := db.NewCollection(CollectionConfig{
```

Comment:
I've been seeing a lot of these changes lately, what is the reasoning behind it? Aesthetics? Go best practices?

Reply:
I just like the shorter names where readability isn't paramount